text/0143-versioning-and-stability.md

```diff
@@ -78,28 +87,39 @@ Code is only marked as deprecated when the replacement becomes stable. Deprecate
 
 ### Removal
 
-Packages are end-of-life’d by being removed, along with a major version bump.
+Packages are end-of-life’d by being removed from the release. The release then makes a major version bump.
 
 We currently have no plans for deprecating signals or creating a major version past v1.0.
 
+For clarity, it is still possible to create a new, backwards incompatible version of an existing type of  signal without actually moving to v2.0 and breaking support. Allow me to explain.
+
+Imagine we develop a new, better tracing API - let's call it AwesomeTrace. We will never mutate the current tracing API into AwesomeTrace. Instead, AwesomeTrace would be added as an entirely new signal which coexists and interoperates with the current tracing signal. This would make adding AwesomeTrace a minor version bump, *not* v2.0. v2.0 would mark the end of support for current tracing, not the addition of AwesomeTrace. And we don't want to ever end that support, if we can help it.
+
+This is not actually a theoretical example. OpenTelemetry already supports two tracing APIs: OpenTelemetry and OpenTracing. We invented a new tracing API, but continue to support the old one.
+
 ## Version Numbers
 
 OpenTelemetry follows [semver 2.0](https://semver.org/) conventions, with the following distinction.
 
-For the purposes of versioning, all code within a release is treated as if it were part of a single package. All packages within a release receive the same version number. By keeping version numbers in lockstep, end users can understand which packages belong to which release.
+OpenTelemetry clients have four components: API, SDK, Semantic Conventions, and Contrib.
+
+For the purposes of versioning, all code within a component is treated as if it were part of a single package, and versioned with the same version number, except for contrib.
 
-Note that contrib packages are released separately from core packages. Contrib releases are not required to match version numbers with the core releases.
+* API packages for all signals version together, across all signals. Signals do not have separate version numbers. There is one version number that applies to all signals that are included in the API release that is labeled with that particular version number.
+* SDK packages for all signals version together, across all signals. There is one version number that applies to all signals that are included in the SDK release that is labeled with that particular version number.
```

Comment:
Should we explicitly point out that these do not necessarily have to match to make it less surprising?

Reply:
good idea